File: ClientApp/bank-client/src/Models/UserDTO.ts
export interface UserDTO {
    name: string;
    surname: string;
    lastname: string;
    birthDate: Date;
    sex: boolean;
    passportSeries: string;
    passportNumber: string;
    issuedBy: string;
    issuedDate: Date;
    passportId: string;
    birthPlace: string;
    residenceCity: number;
    residenceAddress: string;
    maritalStatus: number;
    citizenship: number;
    disability: number;
    isRetiree: boolean;
    monthlyIncome?: number;
    isConscripted: boolean;
    homePhone?: string;
    mobilePhone?: string;
    email?: string;
}

File: BankAPI/DTO/UserDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BankAPI.DTO
{
    public class UserDTO
    {
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Lastname { get; set; }
        public DateTime? BirthDate { get; set; }
        public bool? Sex { get; set; }
        public string PassportSeries { get; set; }
        public string PassportNumber { get; set; }
        public string IssuedBy { get; set; }
        public DateTime? IssuedDate { get; set; }
        public string PassportId { get; set; }
        public string BirthPlace { get; set; }
        public int? ResidenceCity { get; set; }
        public string ResidenceAddress { get; set; }
        public string HomePhone { get; set; }
        public string MobilePhone { get; set; }
        public string Email { get; set; }
        public byte? MaritalStatus { get; set; }
        public byte? Citizenship { get; set; }
        public short? Disability { get; set; }
        public bool? IsRetiree { get; set; }
        public decimal? MonthlyIncome { get; set; }
        public bool? IsConscripted { get; set; }
    }
}
